apps/manager_app/export_wallets.c

```diff
@@ -144,7 +144,17 @@ static void fill_wallet_list(manager_get_wallets_result_response_t *resp) {
     uint8_t wallet_info = get_wallet_info(flash_index);
     list[index].has_passphrase = WALLET_IS_PASSPHRASE_SET(wallet_info);
     list[index].has_pin = WALLET_IS_PIN_SET(wallet_info);
-    list[index].is_valid = (VALID_WALLET == get_wallet_state(flash_index));
+
+    Flash_Wallet *flash_wallet = NULL;
+    get_flash_wallet_by_name(wallet_list[index], &flash_wallet);
+
+    ASSERT(NULL != flash_wallet);
+    if (flash_wallet->is_wallet_locked || VALID_WALLET != flash_wallet->state ||
```

Comment:
can list the types of wallet states that are non-operational. this is only for completeness and later reference.

example locked, partially synced wallet (wallet without device share), invalid wallet, partially deleted wallet, etc.

Reply:
added